manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  NestedInstallerType: nullsoft
+  InstallerUrl: https://update.flipperzero.one/builds/qFlipper/1.3.0/qFlipperSetup-64bit-1.3.0.exe
+  InstallerSha256: 1B4BEF70AD21359E5AA36B6C9E875C9BD158BAD11237F07C91145D752C15D208

```

```suggestion
  InstallerSha256: 1B4BEF70AD21359E5AA36B6C9E875C9BD158BAD11237F07C91145D752C15D208
  ProductCode: qFlipper
```
Add ProductCode for better upgrade matching.